articles/security/token-exp.md

```diff
@@ -0,0 +1,41 @@
+# Token Expiration 
+
+This document states the expiration of different types of tokens that are referenced in Auth0 documentation. [Click here to learn more about the types of tokens.](/tokens)
+
+## Auth0 `access_token`
```

Comment:
`access_token`s issued for user defined APIs will have their expiration define under their settings tab in the dashboard.